.eslintrc.js

```diff
@@ -18,6 +19,12 @@ module.exports = {
   },
   rules: {
     semi: [2, "never"],
-    'no-nonoctal-decimal-escape': 'off'
-  }
+    'no-nonoctal-decimal-escape': 'off',
```

Comment:
`no-nonoctal-decimal-escape` isn't supported in the current eslint version, please double check if this rule is needed; without this rule also fine.